decidim-comments/app/models/decidim/comments/comment.rb

```diff
@@ -112,10 +112,14 @@ def self.export_serializer
       end
 
       def self.newsletter_participant_ids(space)
-        Decidim::Comments::Comment.includes(:root_commentable).not_hidden
-                                  .where("decidim_comments_comments.decidim_author_id IN (?)", Decidim::User.where(organization: space.organization).pluck(:id))
-                                  .where("decidim_comments_comments.decidim_author_type IN (?)", "Decidim::UserBaseEntity")
-                                  .map(&:author).pluck(:id).flatten.compact.uniq
+        # Decidim::Comments::Comment.includes(:root_commentable).not_hidden
```

Comment:
Any reason for commenting this?

Reply:
No @andreslucena we should catch this during review, but you reviewed faster :smile: 
I'm removing it now

Reply:
OK.
Personally, for catching these mistakes before the commit I usually do `git add -p` and make the decision on a block by block basis.